pythonforandroid/bootstraps/sdl2/build/build.py

```diff
@@ -518,6 +527,10 @@ def parse_args(args=None):
                     help='Add this Java class as an Activity to the manifest.')
     ap.add_argument('--activity-launch-mode', dest='activity_launch_mode',
                     help='Set the launch mode of the main activity in the manifest.')
+    ap.add_argument('--save-manifest', dest='save_manifest', default='',
+                    help="If set to a path, the created manifest will be saved to this path.")
+    ap.add_argument('--load-manifest', dest='load_manifest', default='',
+                    help="If set to a path, the manifest will be loaded from this path.")
```

Comment:
Not sure if the paths will be messed up at this point, since the script is run from a directory that is different to the one the user ran from, so e.g. if they wrote './mymanifest.xml' that path won't make sense any more. I think there is some handling of this issue elsewhere in the p4a arguments.

Reply:
that's a good point, we should probably document that it should be an absolute path.